### DISCUSSION TITLE
Uncaught SyntaxError: Unexpected token '(' after minifying js file

### DISCUSSION THREAD

**Discussion by khanakia** [webpack/webpack#12514] [Q&A] [answered]
# Bug report
**What is the current behavior?**
I am simply importing a library `import "bootstrap/dist/js/bootstrap.bundle.min.js"` and when i compile with webpack it works fine but when i **minify the js file** manually or using some wordpress plugin it gives the error `Uncaught SyntaxError: Unexpected token '(' after minifying js file`

**If the current behavior is a bug, please provide the steps to reproduce.**
1. Clone the repo https://github.com/khanakia/webpack-semicolon-issue/
2. Run yarn and yarn install
3. Check this LINE 63 in https://github.com/khanakia/webpack-semicolon-issue/blob/0c6da3e15ed60e04225d0c4766810d00560ae92b/dist/app.js#L63
4. It did not added any **semicolon**

**What is the expected behavior?**
`__webpack_require__.x = x => {}` should be `__webpack_require__.x = x => {};` with semicolon

**Other relevant information:**
webpack version:  ^5.18.0
Node.js version:  15.0
Operating System: OXS
Additional tools: yarn

**Comment by alexander-akait** [comment]
No problems with `Uncaught SyntaxError: Unexpected token '(' after minifying js file`, you example works fine

**Reply by khanakia** [reply]
Please check line 63 https://github.com/khanakia/webpack-semicolon-issue/blob/0c6da3e15ed60e04225d0c4766810d00560ae92b/dist/app.js#L63
There is no semicolon inserted in the end.
It should have semicolon inserted

**Reply by alexander-akait** [reply]
No, it is unnecessary, please read about `ASI`, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#automatic_semicolon_insertion

**Reply by khanakia** [reply]
Yes i know. But i am using webpack with wordpress. We have wordpress plugin which minifies the code and then it throws the error in console because of semicolon. Please check attached screenshots.

<img width="1436" alt="Screenshot 2021-01-27 at 6 12 48 PM" src="https://user-images.githubusercontent.com/2174170/105992668-6a1f5d00-60cb-11eb-97ed-e64f555d8fde.png">
<img width="574" alt="Screenshot 2021-01-27 at 6 12 55 PM" src="https://user-images.githubusercontent.com/2174170/105992686-6f7ca780-60cb-11eb-9b9d-0e31dd451bb9.png">

**Reply by alexander-akait** [reply]
Open an issue in `wordpress plugin`, it is bug in parser on their side, yes, we will add `;`, but it still not webpack problem

**Reply by alexander-akait** [reply]
https://github.com/webpack/webpack/pull/12515

**Reply by khanakia** [reply]
But how is it the plugin issue. Plugin simple minifies the js that's all. It's just a semicolon issue.

**Reply by alexander-akait** [reply]
@khanakia it is valid JS syntax and works everywhere, if some wordpress plugin breaks the core after minimizing, it means they minimizer is broken, and you need repot about this, no webpack problems here

**Reply by khanakia** [reply]
@alexander-akait No worries. Anyway i see you created a pull request #12515 so that should fix the issue :)
Thank you very much.

**Reply by khanakia** [reply]
@alexander-akait I see that the changes are merged into 5.8.0 https://github.com/webpack/webpack/compare/v5.18.0...master
But when i install the webpack using `npm install or yarn` i am still getting old copy and i do not see these changes https://github.com/webpack/webpack/commit/6c5d48246a128029f60f2a17f7672b46e3412ebc

I tried to clean all the cache yarn, npm. You can refer the same repo i shared above.
Please give me some insight what can i do to get the fresh webpack 5.8.0 with all the new commits

**Reply by alexander-akait** [reply]
Because it was not released yet

**Comment by sokra** [comment, marked as answer]
fixed now

**Reply by khanakia** [reply]
Thank you
